apps/web-api/src/collection/collection.service.ts

```diff
@@ -75,7 +75,13 @@ export class CollectionService extends BaseService<Collections, CollectionDTO> {
     this.applyLimitOffset(qb, queryArgs);
     this.applyWhereCondition(qb, queryArgs);
     this.applyOrderCondition(qb, queryArgs);
-    const collections = await qb.getRawMany();
-    return collections;
+    return qb.getRawMany();
+  }
+
+  getByCollectionId(id: number) {
+    return this.find({
+      where: { collection_id: { _eq: id } },
+      limit: 1000, // because default 10
```

Comment:
How I know collection_id is primary key and it's unique. If we use filter by collection_id only one record should be returned. Why we are using limit?

Reply:
I missed it